src/consensus/validation.h

```diff
@@ -54,6 +54,8 @@ enum class TxValidationResult {
     TX_CONFLICT,
     TX_MEMPOOL_POLICY,        //!< violated mempool's fee/size/descendant/RBF/etc limits
     TX_NO_MEMPOOL,            //!< this node does not have a mempool so can't validate the transaction
+    TX_SINGLE_FAILURE,        //!< fee was insufficient to meet some policy, but can change if submitted in a (different) package
```

Comment:
It was not obvious to me what the motivation for `SINGLE` in the constant name here is. After discussing with @glozow, maybe:

```suggestion
    TX_RECONSIDERABLE,        //!< mining score was insufficient to meet some policy, but transaction might be acceptable if submitted in a (different) package
```

Reply:
Yeah renamed, I think it'll help make the connection with `m_reconsiderable_rejects` filter later as well.